### ISSUE TITLE
patch 3652 reclaim collection feature

### ISSUE THREAD

**Issue by Exotic-Retard** [FAForever/fa#1152]
that feature which collects reclaim values when pressing ctrl shift into circles is counterproductive.

it doesn't make it clearer, it makes it not possible to tell which unit is worth more reclaim.
i noticed it changes with zoom level, if so i would really really increase the distance at which they they are collected, or remove the feature entirely.

currently (patch3650) my only issue with the feature is that it doesnt show some rocks, because they have individually low reclaim values, even when together they make a huge difference cos there is a lot of them. in that case i guess this feature could be used.

tldr - this could be useful with proper tuning, but definitely not as i saw it today.

**Comment by aeoncleanse** [comment]
I concur. Having played around with it, I think this change should be flat out reverted. If anything, it should ONLY gather and show stuff from rocks usually below the UI threshold.

**Comment by aeoncleanse** [comment]
Why closed?

**Comment by speed2CZ** [comment]
Can it be changed to show wrecks only where you're currently looking instead of on whole map? And then if you zoom out further it could start collecting them into circles.

Right now you really have to zoom in a lot to see individual wrecks. And without circles and many wrecks it lags my game for up to 10 seconds. So some solution in betweeen would be cool

**Comment by aeoncleanse** [comment]
This is definitely still an issue.

**Comment by Sheeo** [comment]
Uh, I don't believe I closed this but the log says I did, so hm.

Let's just pull the feature from 3652.

**Comment by aeoncleanse** [comment]
Reverted for now

**Comment by Crotalus** [comment]
2c7ca39

Made the reclaim labels UI-moddable now so every user can change the behavior to their liking, so this issue is closed I guess
